packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -649,6 +649,24 @@
   color: var(--base-color-scale-black-0);
 }
 
+.SubdomainNavBar-skip-to-content {
+  position: fixed;
+
+  /* Visually center button (header 75px - button 49px) / 2 = 13px */
+  top: 13px;
+  left: 13px;
```

Comment:
Does it need to be 13px or can you use `var(---base-size-12)` here? We typically avoid hard coding px values and prefer rem.